### DISCUSSION TITLE
One-piece-flow, then Unattended operation, repeat…

### DISCUSSION THREAD

**Discussion by aguerens-2xA** [frePPLe/frepple#660] [Q&A]
Starting to learn Frepple. We have a particular scenario where the assembly line has:
1. A set of continuous operations in a one-piece-flow mode. Each operation takes minutes.
2. Units accumulate to a pre-set number and go together into an oven (unattended operation). This takes hours.
3. When the container comes back from the oven, another set of one-piece-flow op's start, and so forth...
Can this be modeled in the Production module in Frepple?
Thanks for shedding some light.
Alec

**Comment by jdetaeye** [comment]
You may check out https://github.com/frePPLe/frepple/discussions/655    It handles a very similar modeling situation.

A worth of advice;
Trying to modeling everything at a very detailed / physical level can lead to complex and user-unfriendly models.
Production planners typically think at a higher level of abstraction to keep things practical and understandable.   
The planner may for instance think of his production capacity in terms of "I can do X kg per day" or "I can do X units per day".   If the planning tool doesn't work & think at that same level, your planner may find it hard to use.

**Comment by aguerens-2xA** [comment]
Thanks for the fast turnaround, Johan.

I will go through the article you pointed me to and check how it maps out
to our scenario.

Your point on prioritizing volumes over method was duly noted.

Thanks,
-------------------------
Alec Guerenstein
2xA Inc - Co-Founder
***@***.***


On Mon, Jun 17, 2024 at 4:09 AM Johan De Taeye ***@***.***>
wrote:

> You may check out #655
> <https://github.com/frePPLe/frepple/discussions/655> It handles a very
> similar modeling situation.
>
> A worth of advice;
> Trying to modeling everything at a very detailed / physical level can lead
> to complex and user-unfriendly models.
> Production planners typically think at a higher level of abstraction to
> keep things practical and understandable.
> The planner may for instance think of his production capacity in terms of
> "I can do X kg per day" or "I can do X units per day". If the planning tool
> doesn't work & think at that same level, your planner may find it hard to
> use.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/frePPLe/frepple/discussions/660#discussioncomment-9791484>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BE5FNJKJO3CITFC5BUUQMNDZH2KU7AVCNFSM6AAAAABJMBLYICVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOJRGQ4DI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


--
